packages/helm-charts/common/templates/_helpers.tpl

```diff
@@ -430,3 +430,34 @@ prometheus.io/scrape: "true"
 prometheus.io/path:  "{{ $pprof.path | default "/debug/metrics/prometheus" }}"
 prometheus.io/port: "{{ $pprof.port | default 6060 }}"
 {{- end -}}
+
+{{- /* Needs a serviceAccountName in the pod with permissions to access gstorage */ -}}
+{{- define "common.gsutil-sync-data-init-container" -}}
+- name: gsutil-sync-data
+  image: gcr.io/google.com/cloudsdktool/cloud-sdk:latest
+  imagePullPolicy: Always
+  command:
+  - /bin/sh
+  - -c
+  args:
+  - |
+     # If older than upload period, remove the chain data dir.
+     if [ -d /root/.celo/celo/chaindata ]; then
+       mtime=$(stat -f "%Y" /root/.celo/celo/chaindata)
+       day=$(date +%s)
+       diff=$(("$day" - "$mtime"))
+       # If mtime is older than 1 day old, pull the chaindata rather than using the current PVC.
+       if [ "$diff" -gt 86400 ]; then
+         rm -rf /root/.celo/celo
```

Comment:
can you think of any unintended implications to deleting the whole `celo` directory instead of just replacing the `chaindata` dir?
The only thing I can think of that may be useful is the txpool and the persisted discovery table

Reply:
Maybe node keys? I switched into just the chain data